styles/styles.css

```diff
@@ -164,6 +166,15 @@ main .section {
   padding-right: var(--section-padding);
 }
 
+.default-content-wrapper h4 {
```

Comment:
I don't think that this is valid for _all_ H4 elements (I thought i removed it at one point?)